crates/pallet-domains/src/lib.rs

```diff
@@ -2103,6 +2103,15 @@ impl<T: Config> Pallet<T> {
             .map(|domain_object| domain_object.domain_config.runtime_id)
     }
 
+    pub fn domain_id(runtime_id: RuntimeId) -> Option<DomainId> {
+        RuntimeRegistry::<T>::get(runtime_id).and_then(|runtime_object| {
+            runtime_object
+                .raw_genesis
+                .domain_id()
+                .expect("DomainId must be valid since domain is insantiated; qed")
+        })
+    }
```

Comment:
Do we need this method? I don't see it used anywhere.

Actually, we are unable to get domain id by runtime id, because the runtime object is shared by all the domain instances that instantiated with the same runtime id and within the `raw_genesis` the domain id storage is empty. The domain id storage is only set in `into_complete_raw_genesis` when we need the completed raw genesis of a specific domain.